CefSharp.BrowserSubprocess.Core/TypeUtils.cpp

```diff
@@ -30,6 +31,11 @@ namespace CefSharp
             type = obj->GetType();
         }
 
+        if (type == JavascriptObjectWrapper::typeid) {
```

Comment:
At first glance this looks somewhat out of place. Will have to look at in more detail.